airflow/models.py

```diff
@@ -1494,6 +1494,9 @@ def get_template_context(self, session=None):
         prev_execution_date = task.dag.previous_schedule(self.execution_date)
         next_execution_date = task.dag.following_schedule(self.execution_date)
 
+        next_ds = next_execution_date.isoformat()[:10]
```

Comment:
what is the difference between next_ds with tomorrow_ds  and yesterday_ds with prev_ds?

Reply:
next_ds = ds + schedule_interval
tomorrow_ds = ds + 1